content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,424 @@
+
+
+## Contents of an Interchange object
+
+To emphasize a detail above, the key objective of the Interchange project at present is to make it
+easier to get OpenFF force fields in use in non-OpenMM workflows. The scope of this task is large,
+but it is finite; the
+[SMIRNOFF](https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/users/smirnoff.html)
+specification used by OpenFF force fields provides clarity to many questions of information
+content in force fields, which is key for developing an interface to it. The [OpenFF
+Toolkit](https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/topology.html) also already provides
+convenient representations of molecular topologies and functions for parsing existing data into its
+models. Note that this does not close the door on interfacing with non-OpenFF models, it simply
+provides a starting point. Expanding this work outside of the OpenFF stack is discussed below
+("Future work").
+
+An `Interchange` object stores all the information known about a system; this includes its
+chemistry, how that chemistry is represented by a force field, and how the system is organised in 3D
+space. An `Interchange` object has four components, each of which is also an attribute of the core
+Python class
+
+The `topology` attribute stores chemical information in a graph-like structure, without specific information
+about forces from the force field. Currently, this attribute is literally a
+[`Topology`](https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.topology.Topology.html#openff.toolkit.topology.Topology)
+object from the OpenFF Toolkit. This representation includes necessary information for molecular
+mechanics representations (masses, elements, and basic bond information) but also richer chemical
+information used by cheminformatics toolkits and some portions of the SMIRNOFF specification (bond
+orders, bond and atom stereochemistry, aromaticity). While this may change in the future, this does
+mean that coarse-grained representations are not supported in any explicit way.
+
+The `handlers` attribute is a mapping between chemical information and force field parameters as
+they are applied to the system (i.e. post-parametrization). A unique identifier of a segment of the
+chemical graph - for example, the four atoms participating in a dihedral - is mapped to a unique
+identifier of the physical parameters assigned to them - in this example, the force constant, phase,
+and periodicity of the dihedral force. This allows for some de-duplication of parameters; by
+pointing to a representation of the force, each unique parameter is only stored once. The mapping
+between chemical topology and force parameters serves as a way of connecting the forces actually
+present in a simulation to how they were applied by the force field.
+
+Particle positions are stored as a unit-tagged `Nx3` matrix, where `N` is the number of particles in
```

Comment:
I had trouble spotting that positions and box vectors were items 3 and 4 of the "four components" - Maybe some additional formatting could clear this up.